### PR TITLE
Subbranch

### PR DESCRIPTION
**Pull request by Ani-Emmanuel**
To make sure that best practice is followed and the project stays on point i have added this two packages Eslint and standardJS.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -16,9 +17,11 @@ npm i aslang -g
 ```
 
 ## Contributing
+
 Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 
 Please make sure to update tests as appropriate.
 
 ## License
-[MIT](https://choosealicense.com/licenses/mit/)
```
Comment (ezesundayeze) [inline readme.md]: Any particular reason for the space? or is it standardjs that added those spaces?

Comment (ezesundayeze) [reply readme.md]: Every other thing looks good

**Comment by Ani-Emmanuel** [comment]
have you review this

**Review by ezesundayeze — APPROVED** [review]

**Comment by ezesundayeze** [comment]
Not yet. There is a conflict

**Pull request merged**